### PR TITLE
404: International scheduled payment consent details

### PR DESCRIPTION
**Pull request by andra-racovita**
Issue: https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/issues/404

### CODE REVIEW

File: securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/InternationalPaymentConsentDetailsConverterTest.java [hunk 1]
```diff
@@ -37,7 +36,7 @@ public class InternationalPaymentConsentDetailsConverterTest {
     @Test
     public void shouldConvertConsentDetailsToInternationalPaymentConsentDetails() {
         // Given
-        JsonObject consentDetails = aValidInternationalPaymentConsentDetails(INTERNATIONAL_PAYMENT_INTENT_ID);
+        JsonObject consentDetails = aValidInternationalPaymentConsentDetails(INTERNATIONAL_SCHEDULED_PAYMENT_INTENT_ID);
```
Comment (jorgesanchezperez) [inline securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/InternationalPaymentConsentDetailsConverterTest.java:39]: I'm confuse with that change, this has been done in `InternationalPaymentConsentDetailsConverterTest` but the expected implementation test was `InternationalScheduledPayment......`, It seems that we are overriding a existing test class.

File: securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/InternationalScheduledPaymentConsentDetailsConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+/**
+ * Unit test for {@link InternationalScheduledPaymentConsentDetailsConverter}
+ */
+@Slf4j
+public class InternationalScheduledPaymentConsentDetailsConverterTest {
+    @Test
+    public void shouldConvertConsentDetailsToInternationalScheduledPaymentConsentDetails() {
+        // Given
+        JsonObject consentDetails = aValidInternationalScheduledPaymentConsentDetails(INTERNATIONAL_PAYMENT_INTENT_ID);
```
Comment (jorgesanchezperez) [inline securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/InternationalScheduledPaymentConsentDetailsConverterTest.java:39]: That `INTERNATIONAL_PAYMENT_INTENT_ID` seems wrong for that case, I expect `INTERNATIONAL_SCHEDULED_PAYMENT_INTENT_ID`, probably is to reuse the value, but is confuse to read the source.

File: securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/UtilConverter4Test.java [hunk 3]
```diff
@@ -29,6 +29,8 @@ public class UtilConverter4Test {
     public static final String DOMESTIC_SCHEDULED_PAYMENT_INTENT_ID = "PDSC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
     public static final String DOMESTIC_STANDING_ORDER_INTENT_ID = "PDSOC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
     public static final String INTERNATIONAL_PAYMENT_INTENT_ID = "PIC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
+    public static final String INTERNATIONAL_SCHEDULED_PAYMENT_INTENT_ID = "PISC_1c214525-d0c8-4d13-xxx-b812c6fafabe";
```
Comment (jorgesanchezperez) [inline securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/UtilConverter4Test.java:32]: Just it's here the constant expected in the above test!

**Review by jorgesanchezperez — CHANGES_REQUESTED** [review]
Please review the request changes.

File: securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/UtilConverter4Test.java (hunk 3 above)
Comment (andra-racovita) [reply securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/UtilConverter4Test.java:32]: I made a mix-up between the Intent Ids for International Scheduled Payments and International Payments.

**Review by jorgesanchezperez — APPROVED** [review]

**Pull request merged**